Resources/ServerInfo/Rules.txt

```diff
@@ -13,70 +13,67 @@
-- Your characters should generally hold no more than two departmental jobs and a service side job in-character. You may swap departments from time to time so long as it doesn't happen constantly.
-- Permanent death is not required, as there's off-site DNA storages for all crewmembers.
-
-[color=#7ade00]16.[/color] Stay within your job, reasonable knowledge, and abilities. Referred to as metagaming.
-- Unless your character has a past in the type of work they shouldn't know how to do jobs outside of theirs.
-- Do not do other people's jobs for them, only play the role you signed up for. If you want to play a different job ask the Head of Personnel for a job change. Exceptions can be made for if no active staff exist for a department.
-- Basic crew should not be knowledgeable on most non-NT technology and items, unless they've experienced it in-character. Security and command may have some leeway within reason.
-
-[color=#7ade00]17.[/color] Command and security are expected to be competent, lawful, and follow procedure.
-- Neither roles should be grossly incompetent at their job or in their judgment. You are held to higher standards and are usually looked up to be good at your work, a role model, and reflection of your department, act like it.
-- You shouldn't be knowingly breaking the law consistently as a command or security member.
-- Command staff should know Standard Operating Procedure and follow it to the best of their abilities. Security should know both Standard Operating Procedure, Space Law, follow it, and enforce it to the best of their abilities. Failure to do so is likely to result in role bans.
-- (We currently use Wizard's Den Space Law and SOP you can find those at https://wiki.spacestation14.io/wiki)
\ No newline at end of file
+- If you enter a dead state and get cloned back to life you forget all the events since the beginning of the shift.
+- Respawning as a new character eliminates all memory of everyhting that may have happened previously in that round.
+
+[color=#7ade00]15.[/color] Full continuity of your character and events that happen to them is to be expected. All rounds are canonical unless stated otherwise by an admin.
+- You are allowed to recall past shifts in any manner unless it violates the New Life rule.
+- Punishments you receive will be enforced and carried out in-character throughout rounds. If your character gets demoted, you are expected to stay out of that job until a reasonable amount of time has passed or you get rehired again in-game.
+- Your characters should generally hold no more than two departmental jobs and a service side job in-character. You may swap departments from time to time so long as it doesn't happen constantly or consistently.
+- Permanent death is not required, as there is an off-site DNA storages available for all crewmembers.
+
+[color=#7ade00]16.[/color] Stay within your job, reasonable knowledge, and abilities. (also referred to as metagaming).
```

Comment:
My read of "Stay within your job, reasonable knowledge, and abilities (also referred to as metagaming)" implies that staying within your job would be metagaming.